subprojects/docs/src/docs/userguide/dep-man/dependency_management.adoc

```diff
@@ -21,6 +21,7 @@ include::dependency_version_alignment.adoc[leveloffset=+1]
 include::dependency_capability_conflict.adoc[leveloffset=+1]
 include::component_metadata_rules.adoc[leveloffset=+1]
 include::resolution_rules.adoc[leveloffset=+1]
+include::consistent_resolution.adoc[leveloffset=+1]
```

Comment:
:x: This also needs to be added to the "nav bar" in "header.html".

I also would put it before "resolution_rules.html". Because resolution_rules describes the "last resort" features, while this is a feature we encourage you to use.

Or add the _consistent resolution_ topic to the "Preventing accidental dependency upgrades" section instead of an additional page. It's kind of the same topic, isn't it?

Reply:
Yeah I put it last because it's incubating, but that's not necessarily a problem.

Reply:
Merged with the "tuning" docs